oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DefaultScopeDefine.java

```diff
@@ -101,6 +103,7 @@
     public static final String SERVICE_RELATION_CATALOG_NAME = "SERVICE_RELATION";
     public static final String SERVICE_INSTANCE_RELATION_CATALOG_NAME = "SERVICE_INSTANCE_RELATION";
     public static final String ENDPOINT_RELATION_CATALOG_NAME = "ENDPOINT_RELATION";
+    public static final String PROCESS_CATALOG_NAME = "PROCESS";
```

Comment:
You should update the process catalog, although we don't set the alert rules of process for now, but most likely we will, and also auto profiling should sent the notification(we need to extend alarm module until then)

Reply:
Adapted to the alarm module. But the `ProcessTraffic` only works for the registration mechanism, don't need to implement the `WithMetadata` for the alarm module. Also, because we use a hash algorithm(sha512) to generate the process id, we can't analyze the metadata in the process, so I add a new Entity to appoint them.